packages/resource-deployment/templates/batch-account-dev.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "scanRequestPoolNodes": {
```

Comment:
we are depreciating legacy workflow and can drop `scanRequestPoolNodes` and `urlScanPoolNodes` completely

Reply:
@lamaks , we can drop all the legacy stuff in another pr. 
(if we just remove it, we need to also remove the job schedule that depends on them etc)